src/core/Form/LabelText/LabelText.baseStyles.tsx

```diff
@@ -0,0 +1,18 @@
+import { css } from 'styled-components';
+import { LabelTextProps } from './LabelText';
+import { withSuomifiTheme, TokensAndTheme } from '../../theme';
+import { font } from '../../theme/reset';
+import { Omit } from '../../../utils/typescript';
+
+export const baseStyles = withSuomifiTheme(
+  ({ theme }: TokensAndTheme & Omit<LabelTextProps, 'children'>) => css`
```

Comment:
Are there any need to Omit anything as there seems to be no props in use from the `LabelTextProps` so it might be dropped totally?

Reply:
I'll check if it works.

Reply:
Probably legacy since it had the fullWidth prop in use.

Reply:
Seems to work without it as well, removed.